consensus/qbft/src/test/java/org/hyperledger/besu/consensus/qbft/QbftBlockHeaderValidationRulesetFactoryTest.java

```diff
@@ -60,7 +61,8 @@ public void bftValidateHeaderPasses() {
             .buildHeader();
 
     final BlockHeaderValidator validator =
-        QbftBlockHeaderValidationRulesetFactory.blockHeaderValidator(5, false).build();
+        QbftBlockHeaderValidationRulesetFactory.blockHeaderValidator(5, false, Optional.empty())
```

Comment:
nit: Similar to the IbftBlockHeaderValidationRulesetFactoryTest can you add a method in the test to create the blockHeaderValidator, or maybe just a field variable